### PR TITLE
Add names to extra Trident and Amplio2 units

### PR DESCRIPTION
**Pull request by petervdmeer**
New Trident units were added around 2019/2020, but many of them did not have a name yet. Added names for those units (based on the discussions on the forum around that time).

Some units have both a "functional name" as well as a "historical type/manufacturer name". I personally prefer to refer to units using the "function name" in my own Rulesets, but Rulesets that try to be historically accurate will prefer the type/manufacturer name.

### CODE REVIEW

**Comment by cazfi** [comment]
First of all: Sorry for the delay in handling this. Our ticket tracker, required for handling any patch merges, at osdn has had issues for some time now, and it has taken some time to get replacement to use. There is progress...

As for the patch itself, could you combine the commits and force push it to this PR branch? We are going to want just one commit of this in the git history.

**Comment by petervdmeer** [comment]
No problem. I combined the commits and also added relevant names for new Amplio2 units (which also didn't have a name yet).

File: data/amplio2/units.spec [hunk 1]
```diff
@@ -52,7 +52,7 @@ tiles = { "row", "column", "tag"
   0,  1, "u.howitzer_Idle"      ; [Nemo]
   0,  2, "u.battleship_Idle"    ; [Nemo]
   0,  3, "u.bomber_Idle"        ; [GB]
-  0,  4, "u.cannon_Idle"        ; [CT]
+  0,  4, "u.early_cannon_Idle"  ; [CT]
```
Comment (cazfi) [inline data/amplio2/units.spec]: Changes like this are a bit more complicated. The main units gfx file should have all the standard units (those that oure included rulesets use). "extra_units" are just for more custom units.

So, if you want sprite from extra_units to become the official cannon, you need to switch the sprites in the .png -files instead.

**Review by cazfi — CHANGES_REQUESTED** [review]

File: data/amplio2/units.spec (hunk 1 above)
Comment (petervdmeer) [reply data/amplio2/units.spec]: I swapped some images in the trident set, and restored the original Amplio2 set.

**Comment by cazfi** [comment]
Ticket to handle this: https://redmine.freeciv.org/issues/161

**Comment by cazfi** [comment]
Merged outside github UI.

**Pull request closed without merging**